src/core.js

```diff
@@ -750,6 +750,12 @@ define([
 
         $modalBody.append($(find_usages({tableData: tableData})));
 
+        $modalBody.find('.link-to-question').click(function() {
```

Comment:
This event handler function should return false. Otherwise I think it adds a `#` to the URL in the address bar, which is annoying.